src/coord.jl

```diff
@@ -83,6 +86,17 @@ function Transformation(
     return Transformation(pj, direction)
 end
 
+function Transformation(
+    pipeline::AbstractString,
+    always_xy::Bool = false,
+    direction::PJ_DIRECTION = PJ_FWD,
```

Comment:
Why is `area::Ptr{PJ_AREA} = C_NULL` not included like in the other constructors? Or does that not apply to pipelines?

Reply:
Area is used to restrict the search for a pipeline between two CRS. Here we directly construct a pipeline, so no search is involved.